### DISCUSSION TITLE
Optional type

### DISCUSSION THREAD

**Discussion by hanako-eo** [slint-ui/slint#5250] [Ideas] [answered]
Hello, I start to use Slint in a project but I try to found if in the `.slint` files we have the optional type (to do something like this `in-out property <string?> file_path = null;`) and I found nothing. I open this discussion to know a little bit more about that. (I know with my example I can use an empty string instead of null but I prefer to use semantically null or none)

**Comment by ghost** [comment, marked as answer]
Hi, thank you for your feedback. There is an issue for that topic. Currently there is no optional type in Slint. Here you can find the ticket #5164.
